### DISCUSSION TITLE
Ignore user replies in streaming?

### DISCUSSION THREAD

**Discussion by flower-pixel** [tweepy/tweepy#1551] [Questions] [answered]
edit:  nm  I'll just ignore status that starts with '@'

**Comment by ThomasH-Dev** [comment]
Could I ask how you specifically filter out certain words?

**Reply by Harmon758** [reply]
That's not really relevant to this discussion, but the standard streams don't support this.
You'll have to filter them out yourself by checking the text of each Tweet.

**Comment by Harmon758** [comment, marked as answer]
You can check the `in_reply_to_status_id`, `in_reply_to_status_id_str`, `in_reply_to_user_id`, `in_reply_to_user_id_str`, or `in_reply_to_screen_name` attribute of the Status/[Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) to determine if it's a reply or not.
